sympy/matrices/matrices.py

```diff
@@ -4224,135 +4228,3 @@ def _find_reasonable_pivot_naive(col, iszerofunc=_iszero, simpfunc=None):
                 return i, tmp_col_val, False, newly_determined
 
     return indeterminates[0][0], indeterminates[0][1], True, newly_determined
-
-class _MinimalMatrix(object):
```

Comment:
What's the reason to delete the classes? I think that @siefkenj designed these recently to better organize the matrix code.

Reply:
Code duplicate of `sympy/matrices/common.py`. The other code appears to be the one that's being used.